services/core-web/src/components/mine/Permit/MinePermitTable.tsx

```diff
@@ -709,19 +752,11 @@ export const MinePermitTable: React.FC<RouteComponentProps & MinePermitTableProp
         recordDescription: "amendment history",
         getDataSource: amendmentHistory,
         subTableColumns: childColumns,
-        expandedRowKeys: props.expandedRowKeys,
-        onExpand: props.onExpand,
+        expandedRowKeys: expandedRowKeys,
+        onExpand: onExpand,
       }}
     />
   );
 };
 
-const mapStateToProps = (state) => ({
-  partyRelationships: getPartyRelationships(state),
-  permitStatusOptionsHash: getDropdownPermitStatusOptionsHash(state),
-  permitAmendmentTypeOptionsHash: getPermitAmendmentTypeOptionsHash(state),
-});
-
-export default withRouter(
-  connect(mapStateToProps)(MinePermitTable) as FC<MinePermitTableProps & RouteComponentProps>
-);
+export default withRouter(MinePermitTable as FC<MinePermitTableProps & RouteComponentProps>);
```

Comment:
I'm wondering, since you changed it to use useHistory and useParams, if it needs the RouteComponentProps still or if it can just be `export default MinePermitTable`

Reply:
Oh, probably.